kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -125,7 +126,8 @@
       numQuestions: 'Number of questions',
       enterNum: 'Enter a number',
       examRequiresTitle: 'The exam requires a title',
-      examRequiresNum: 'The exam requires a number of questions between 1 and 50',
+      numQuestionsBetween: 'The exam requires a number of questions between 1 and 50',
```

Comment:
That might confuse users of why the sudden change in max num of questions. I updated the  error message. @khangmach approved.